tests/stress/compute_flow_with_threads.py

```diff
@@ -200,19 +178,19 @@
     DATA_asset = ocean.assets.resolve(DATA_did)
     ALGO_asset = ocean.assets.resolve(ALGO_did)
 
-    compute_service = get_first_service_by_type(DATA_asset, "compute")
-    algo_service = get_first_service_by_type(ALGO_asset, "access")
+    compute_service = DATA_asset.services[0]
+    algo_service = ALGO_asset.services[0]
 
     environments = ocean.compute.get_c2d_environments(compute_service.service_endpoint)
 
     # Pay for dataset
     DATA_order_tx_id = ocean.assets.pay_for_service(
         asset=DATA_asset,
         service=compute_service,
-        consume_market_order_fee_address=bob_wallet.address,
+        consume_market_order_fee_address=wallet.address,
         consume_market_order_fee_token=DATA_datatoken.address,
         consume_market_order_fee_amount=0,
-        wallet=bob_wallet,
+        wallet=wallet,
```

Comment:
Actually, even if we double-assign this variable, I'd rather have relevant namings. i.e. consumer_wallet. That is, if for the purposes of this test, the publisher_wallet and the consumer_wallet are the same, I'd rather have `consumer_wallet = publisher_wallet = Wallet(...)` than risk confusion. Or, whatever: `alice_wallet`, `bob_wallet`. `wallet` is not a good variable name because it is vague.